x-pack/plugins/global_search_bar/public/plugin.test.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { coreMock } from '@kbn/core/public/mocks';
+import { GlobalSearchBarPlugin, type GlobalSearchBarPluginStartDeps } from './plugin';
```

Comment:
```suggestion
import { GlobalSearchBarPlugin } from './plugin';
import { globalSearchPluginMock } from '@kbn/global-search-plugin/public/mocks';
```

Reply:
Thanks for the suggestion, I wasn't aware we had this mock!